### PR TITLE
Use FQCNs

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request uses FQCNs for modules listed in Ansible tasks.

## 💭 Motivation and context ##

FQCNs for modules are the future of Ansible.

## 🧪 Testing ##

All pre-commit hooks pass.  I also built a staging AMI and deployed it to our COOL staging environment and verified that it continued to behave as expected.

## ✅ Pre-approval checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All new and existing tests pass.
* [x] Build a staging AMI
* [x] Test staging AMI

## ✅ Post-merge checklist ##
* [x] Build a production AMI

### CODE REVIEW

File: src/nessus.yml [hunk 1]
```diff
@@ -4,13 +4,7 @@
   become: yes
   become_method: sudo
   roles:
+    - role: more_ephemeral_ports
     - role: nessus
       vars:
         package_bucket: "{{ build_bucket }}"
-
-- hosts: all
-  name: Configure maximum number of ephemeral ports
```
Comment (mcdonnnj) [inline src/nessus.yml:12]: If we're consolidating blocks then I think the name for the (above) block should be updated to reflect that change.

**Review by mcdonnnj — APPROVED** [review]
I guess the skeleton update handled much of the changes here. What remains looks good to me ✔with my one comment.

**Review by dav3r — APPROVED** [review]
LGTM! 🍰  
Also, I agree with @mcdonnnj's comment.

File: src/nessus.yml (hunk 1 above)
Comment (jsf9k) [reply src/nessus.yml:12]: See commit fbd3023.

**Pull request merged**